packages/components/src/flex/flex/hook.ts

```diff
@@ -62,7 +62,7 @@ export function useFlex( props: WordPressComponentProps< FlexProps, 'div' > ) {
 
 	const classes = useMemo( () => {
 		const base = css( {
-			alignItems: isColumn ? 'normal' : align,
+			alignItems: align ?? ( isColumn ? 'normal' : 'center' ),
```

Comment:
This change definitely makes sense, but I'm not sure if the new behaviour of `HStack` is 100% correct either.

I was taking a look at `HStack` when `direction="column"` , and I was playing around with the `alignment` prop:
- on `trunk`, in this case changing the `alignment` prop doesn't seem to make any differences
- on this PR, it seems to work as expected, except for the `alignment="edge"` case - the items appear to be centered, but I believed they should be stretched (especially given the description of the `edge` value: "Aligns content to the edges of the container.")

<details>

<summary>Looks like this change would fix it</summary>

```diff
diff --git a/packages/components/src/h-stack/utils.ts b/packages/components/src/h-stack/utils.ts
index b1413ee2dc..60151b00a2 100644
--- a/packages/components/src/h-stack/utils.ts
+++ b/packages/components/src/h-stack/utils.ts
@@ -28,7 +28,7 @@ const V_ALIGNMENTS: Alignments = {
 	bottomLeft: { justify: 'flex-end', align: 'flex-start' },
 	bottomRight: { justify: 'flex-end', align: 'flex-end' },
 	center: { justify: 'center', align: 'center' },
-	edge: { justify: 'space-between', align: 'center' },
+	edge: { justify: 'space-between', align: 'stretch' },
 	left: { justify: 'center', align: 'flex-start' },
 	right: { justify: 'center', align: 'flex-end' },
 	stretch: { align: 'stretch' },

```
</details>

Reply:
If I remember properly, I based the behavior of "edge" here by comparing the behavior of "h" in HStack and applied the same behavior but in the other axis.

Reply:
As much as I don't love the arbitrariness of this `align` prop, I agree with Riad that this new `edge` behavior for VStack is correct/consistent within the existing logic of the system.

In this system, I understand `stretch` to apply only to the cross axis, and `edge` to apply only to the main axis.

How about we tweak the description to make that clearer?

```diff
-	 * * `edge`: Aligns content to the edges of the container.
+	 * * `edge`: Justifies content to be evenly spread out up to the main axis edges of the container.
-	 * * `stretch`: Stretches content to the edges of the container.
+	 * * `stretch`: Stretches content to the cross axis edges of the container.
```

Reply:
Thank you both.

Given that the separation between breaking change and bug fix can be blurry here, I agree with Lena's proposed solution